WheelOfFortune/Prompt.cs

```diff
@@ -39,7 +39,16 @@ public static char AddAdditionalPlayer()
         /// </summary>
         public static void StartTurn(Player currentPlayer, Puzzle currentPuzzle)
         {
+            int spacerCount = currentPuzzle.PuzzleSoFar.Length + 20;
+            string spacer = new string('-', spacerCount);
+            string buffer = new string(' ', 9);
```

Comment:
Would it be worth it to add the spacer and buffer strings as properties to the Prompt class if they will be reused? I think I saw the spacer string in the welcome message method too

Reply:
the spacer for this portion is dependent on the length of the puzzle that is passed in, so I think it might make sense to stay here